gpflow/kernels.py

```diff
@@ -1005,6 +1006,94 @@ def Kdiag(self, X, presliced=False):
         return reduce(tf.multiply, [k.Kdiag(X) for k in self.kernels])
 
 
+class ChangePoints(Combination):
+    """
```

Comment:
```suggestion
    r"""
```
Turn it into a raw string when using LaTeX commands (`\cite` below) to ensure the `\c` does not get interpreted as a special character (e.g. `\t` = tab, `\n` = newline).

Reply:
Good catch